src/v2/Apps/Order/Components/__tests__/SavedAddresses.jest.tsx

```diff
@@ -4,12 +4,51 @@ import { setupTestWrapper } from "v2/DevTools/setupTestWrapper"
 import { Button } from "@artsy/palette"
 import { AddressModal } from "v2/Apps/Order/Components/AddressModal"
 import React from "react"
+import { createTestEnv } from "v2/DevTools/createTestEnv"
+import { RootTestPage } from "v2/DevTools/RootTestPage"
+import { userAddressMutation } from "v2/Apps/__tests__/Fixtures/Order/MutationResults"
+import { SavedAddressItem } from "v2/Apps/Order/Components/SavedAddressItem"
 
 jest.unmock("react-relay")
 
-beforeEach(() => {
-  afterEach(() => {
-    jest.clearAllMocks()
+class SavedAddressesTestPage extends RootTestPage {
+  async selectEdit() {
+    this.find(`[data-test="addressModal"]`).simulate("click")
+    await this.update()
+  }
+}
+
+describe("Saved Addresses mutations", () => {
+  const { mutations, buildPage } = createTestEnv({
+    Component: (props: any) => (
+      <SavedAddressesFragmentContainer inCollectorProfile {...props} />
+    ),
+    defaultData: userAddressMutation,
+    TestPage: SavedAddressesTestPage,
+    query: graphql`
+      query SavedAddresses_Test_Query {
+        me {
+          ...SavedAddresses_me
+        }
+      }
+    `,
+  })
+
+  it("edits the saved addresses after calling edit address mutation", async () => {
+    const page = await buildPage()
+    const editButton = page.find(`[data-test="editAddress"]`).first()
+    editButton
+      .props()
+      .onClick(userAddressMutation.me.addressConnection.edges[0].node as any)
+    const addresses = page.find(SavedAddressItem).first().text()
+
+    setTimeout(() => {
+      expect(mutations.mockFetch).toHaveBeenCalledTimes(1)
```

Comment:
suggestion: 
hmm, I didn't get where the 'save' button on the modal is being clicked...
It is good to verify the mutation name and the arguments passed to `mockFetch` with `toHaveBeenCalledWith`

Reply:
I had to time. box these tests, I think a mob session might be a good use of time to address this and perhaps to add a few more.